generators/app/templates/scripts/_karma.conf.js

```diff
@@ -1,62 +1,108 @@
-/* eslint-disable no-console */
+/* eslint-disable no-console, camelcase */
 const serveStatic = require('serve-static');
-const path = require('path');
-const serve = serveStatic(
-  path.join(__dirname, '..'),
-  {index: ['index.html', 'index.htm']}
-);
+const pkg = require('../package.json');
 
+/* allow static files to be served, note karma will takeover the /test directory */
 const StaticMiddlewareFactory = function(config) {
-  console.log(`**** Dev server started at http://${config.listenAddress}:${config.port}/ *****`);
+  console.log(`**** static file server started for ${config.basePath} *****`);
+
+  const serve = serveStatic(
+    config.basePath,
+    {index: ['index.html', 'index.htm']}
+  );
 
   return function(req, res, next) {
     res.setHeader('Cache-Control', 'no-cache,must-revalidate');
     return serve(req, res, next);
   };
 };
 
-module.exports = function(config) {
-  const detectBrowsers = {
-    enabled: false,
-    usePhantomJS: false
-  };
+/* browsers to run on teamcity */
+const teamcityLaunchers = {
```

Comment:
this seems a confusing name given that these browsers are set up to work with browserstack.

Reply:
yeah it is maybe we should split it into `teamcityLaunchers`, and `browserstackLaunchers`

Reply:
yeah, I would just call these browserstackLaunchers and just use it in TC usecase below or something.